### PR TITLE
style.css: Change cursor on tabs and cards

### PR DESCRIPTION
**Pull request by ankit01ojha**
Fixed https://github.com/coala/landing-frontend/issues/16

### CODE REVIEW

**Comment by ankit01ojha** [comment]
I didn't found any problem regarding the tabs( cursor was changing to pointer ), so I only changed regarding the cards.

File: resources/css/style.css [hunk 1]
```diff
@@ -230,6 +230,11 @@ textarea.materialize-textarea {
 .card-title {
   overflow-wrap: break-word !important;
 }
+.card{
+
```
Comment (hemangsk) [inline resources/css/style.css]: new line not required 👍

**Comment by gkrishnan724** [comment]
@hemangsk  This change creates pointer on snippets is that fine?

**Comment by hemangsk** [comment]
@gkrishnan724  @ankit01ojha True, it won't be right to have pointers on snippets as clicking them has no effect, pointers will be needed on project cards and mentor cards

**Comment by ankit01ojha** [comment]
@hemangsk That means I have to make changes individually.

**Comment by hemangsk** [comment]
@ankit01ojha yeah, It'll be not required on the mentor card as we have a clickable github button in the mentor cards, so its only required in the small cards with bear description, ie, ``` .showcase-card```

**Comment by ankit01ojha** [comment]
Thanks:)

**Comment by hemangsk** [comment]
@rultor merge

**Comment by hemangsk** [comment]
🎉

**Pull request merged**